gratipay/billing/exchanges.py

```diff
@@ -198,12 +198,20 @@ def get_ready_payout_routes_by_network(db, network):
                  ) > 0
 
 
-                OR -- Include green-lit Gratipay 1.0 balances
+                OR -- Include team members
+
+                (SELECT count(*)
+                   FROM current_takes ct
```

Comment:
Maybe we should just check `takes` rather than `current_takes`? i.e. whether the user has *ever* been a part of a team